### ISSUE TITLE
Wait for adb timeout is too short

### ISSUE THREAD

**Issue by gabidavid** [JDFind/spoon#1]
Some machines execute adb commands in more than 5s (which, **I suppose**, is the default timeout for running adb commands ) and an exception is thrown:

```
E/adb: Unable to obtain result of 'adb version'
Exception in thread "main" java.lang.NullPointerException
       at com.squareup.spoon.SpoonUtils.waitForAdb(SpoonUtils.java:153)
       at com.squareup.spoon.SpoonUtils.initAdb(SpoonUtils.java:122)
       at com.squareup.spoon.SpoonRunner.run(SpoonRunner.java:118)
       at com.squareup.spoon.Main.main(main.kt:37)
```

In my case, the timeout was triggered by _adb version_, which takes about 7 seconds to execute.

**Comment by JDFind** [comment]
The default adb timeout is defined as 10 minutes.
If it is a timeout issue, maybe you're calling the runner with a smaller value.
Check out the `--adb-timeout <ADB_TIMEOUT>` parameter.

**Comment by JDFind** [comment]
It seems that the default timeout in `AndroidDebugBridge` is indeed 5 seconds and the value is overwritten with `adb-timeout` only after a test run is started.
Will fix, thanks.

**Comment by JDFind** [comment]
Fixed in 540b366
Please re-open if the problem re-appears.
